### PR TITLE
[SDP-821] misc: add XLM asset support

### PR DESCRIPTION
**Pull request by CaioTeixeira95**
### What

Add support to disburse the Stellar Network native token - XLM. Also, add it to the asset default list.

### Why

The SDP should be able to disburse XLM.

### Known limitations

N/A

#### PR Structure

* [x] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [x] This PR does not mix refactoring changes with feature changes (split into two PRs otherwise).
* [x] This PR's title starts with the name of the package, area, or subject affected by the change.

#### Thoroughness

* [x] This PR adds tests for the new functionality or fixes.
* [x] This PR contains the link to the Jira ticket it addresses.

#### Configs and Secrets

* [x] No new CONFIG variables are required -OR- the new required ones were added to the helmchart's [`values.yaml`] file.
* [x] No new CONFIG variables are required -OR- the new required ones were added to the deployments ([`pr-preview`], [`dev`], [`demo`], `prd`).
* [x] No new SECRETS variables are required -OR- the new required ones were mentioned in the helmchart's [`values.yaml`] file.
* [x] No new SECRETS variables are required -OR- the new required ones were added to the deployments ([`pr-preview secrets`], [`dev secrets`], [`demo secrets`], `prd secrets`).

#### Release

* [x] This is not a breaking change.
* [x] **This is ready for production.**. If your PR is not ready for production, please consider opening additional complementary PRs using this one as the base. Only merge this into `develop` or `main` after it's ready for production!

#### Deployment

* [ ] Does the deployment work after merging?

[`values.yaml`]: ../helmchart/sdp/values.yaml
[`pr-preview`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/stellar-disbursement-platform/backend-helm-values
[`dev`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/backend-helm-values
[`demo`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-backend-helm-values
[`pr-preview secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/common-previews/externalsecrets-common-previews.yaml#L241-L346
[`dev secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/stellar-disbursement-platform-externalsecrets.yaml
[`demo secrets`]: https://github.com/stellar/kube/blob/d3e4f5dd8aa4c13b45a31a5a937f3e98841171a7/kube001-dev/namespaces/stellar-disbursement-platform/demo/demo-sdp-externalsecrets.yaml

### CODE REVIEW

**Review by ziyliu — APPROVED** [review]
🚀

File: dev/docker-compose-sdp-anchor.yml [hunk 1]
```diff
@@ -133,6 +133,22 @@ services:
                 "max_amount": 10000
               },
               "withdraw": {"enabled": false}
+            },
```
Comment (marwen-abid) [inline dev/docker-compose-sdp-anchor.yml:136]: Do we want to add XLM to `values.yaml` and `docker-compose-e2e-tests.yml` as well ?

**Review by marwen-abid — APPROVED** [review]
LGTM! Thank you for adding that!

File: dev/docker-compose-sdp-anchor.yml (hunk 1 above)
Comment (CaioTeixeira95) [reply dev/docker-compose-sdp-anchor.yml:136]: We do want to add it to the `values.yaml`. But for the `docker-compose-e2e-tests.yml`, I don't think so. Thank you for pointing that out.

Comment (CaioTeixeira95) [reply dev/docker-compose-sdp-anchor.yml:136]: changed: [afa7ef0](https://github.com/stellar/stellar-disbursement-platform-backend/pull/1/commits/afa7ef088e18ea3d58a677a2a3ff2bf06b2720d8)

**Pull request merged**